CalendarFXView/src/main/java/com/calendarfx/model/EntryEditParameter.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (C) 2017 Dirk Lemmermann Software & Consulting (dlsc.com)
+ * <p>
+ * This file is part of CalendarFX.
+ */
+package com.calendarfx.model;
+
+import com.calendarfx.view.DateControl;
+
+/**
+ * Class used for parameter of {@link com.calendarfx.view.DateControl#entryEditPolicy} functional interface
+ */
+public final class EntryEditParameter {
+
+    /**
+     * The date control the entity is associated with
+     */
+    private final DateControl dateControl;
+
+    /**
+     * The entity the operation is operated on
+     */
+    private final Entry<?> entry;
+
+    /**
+     * The operation
```

Comment:
Javadocs will be displayed inside HTML pages as normal text. Normal text should end with a period ".".